src/midhandshake_dtls_steam.rs

```diff
@@ -35,9 +35,9 @@ where
     /// This corresponds to [`SSL_do_handshake`].
     ///
     /// [`SSL_do_handshake`]: https://www.openssl.org/docs/manmaster/man3/SSL_do_handshake.html
-    pub fn handshake(self) -> Result<DtlsStream<S>, HandshakeError<S>> {
+    pub fn handshake(self) -> Result<SyncDtlsStream<S>, HandshakeError<S>> {
```

Comment:
Though, how are we going to use this for async now?

Reply:
I am not sure we can support this easily since tokio_openssl does not wrap MidHandshakeSslStream